client/app/account/billing/autoRenew/common/billing-autoRenew-common.html

```diff
@@ -120,12 +120,14 @@
             data-text="{{:: 'autorenew_service_HOSTING_PRIVATE_DATABASE_terminate' | translate }}"></oui-action-menu-item>
 
         <oui-action-menu-item
-            data-ng-if="canDisableAutorenew(service) && service.serviceType !== 'EMAIL_DOMAIN'"
+            data-ng-if="canDisableAutorenew(service)"
             data-on-click="disableAutorenew(service)"
             data-text="{{:: 'autorenew_service_disable_button' | translate }}"></oui-action-menu-item>
 
         <oui-action-menu-item
-            data-ng-if="canEnableAutorenew(service) && service.serviceType !== 'EMAIL_DOMAIN'"
+            data-ng-if="canEnableAutorenew(service)
+                        && service.serviceType !== 'EMAIL_DOMAIN'
```

Comment:
```suggestion
                        && service.serviceType !== 'EMAIL_DOMAIN' && && service.serviceType !== 'EXCHANGE'
```

?

Reply:
No need to as `EXCHANGE` services have `renew.automatic: true` (whereas `canDisableRenew` has to be patched because of `renewalType` or `renew.forced` which is sometimes wrong)